spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -140,6 +140,12 @@ sealed class SpdxExpression {
     fun isValid(strictness: Strictness = Strictness.ALLOW_CURRENT): Boolean =
         runCatching { validate(strictness) }.isSuccess
 
+    /**
+     * Return true if this expression offers a license choice. This is only true, if this expression contains the
+     * [OR operator][SpdxOperator.OR].
+     */
+    open fun offersChoice(): Boolean = false
```

Comment:
How about naming it: `containsLicenseCoice`, `containsChoice` or `needsChoice` ?

Reply:
I like `offer`.